### PR TITLE
feat: implement editable points of interest

### PR DESCRIPTION
**Pull request by donni106**
- add active style for sidebar navigation entries
- added logics in controller
- added views
- refactored price informations which were used at events only before, to use it for points of interest as well now
  - needed to adjust js for nested has_many functionality
- formatted all <<~GRAPHQL heredocs because syntax highlightings were broken
  - params with #{...} needs to be on a single line
- adjusted margins and `<hr />`s
- handle validation for operation_company :name

#16

---

Bildschirmfoto am 2021-02-03 um 20 22 52-fullpage

### CODE REVIEW

File: app/views/point_of_interests/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<%= fields_for :point_of_interest, point_of_interest do |f|  %>
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Name *</label>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
```
Comment (donni106) [inline app/views/point_of_interests/_form.html.erb:10]: - [ ] irgendwo muss der data provider hidden mit stehen, sonst fliegt der beim editieren raus

**Review by marcometz — APPROVED** [review]

**Comment by donni106** [comment]
@digorath:

> logo image file for POIs is missing

- [ ] media content of type logo for point of interests
Bildschirmfoto 2021-02-05 um 17 07 00

**Comment by donni106** [comment]
- [ ] handle operating company :name required

**Comment by donni106** [comment]
> @digorath:
> 
> > logo image file for POIs is missing
> 
> * [x]  media content of type logo for point of interests
>   Bildschirmfoto 2021-02-05 um 17 07 00

```
"mediaContents": [
  {
    "contentType": "image",
    "sourceUrl": {
      "url": "http://backoffice2.reiseland-brandenburg.de/rpcServer/file/get/id/336458"
    }
  },
  {
    "contentType": "logo",
    "sourceUrl": {
      "url": "http://backoffice2.reiseland-brandenburg.de/rpcServer/file/get/id/51140"
    }
  }
],
```

there will be a "logo" content type for media contents possible through CMS

**Pull request merged**